dockers-install.sh

```diff
@@ -0,0 +1,72 @@
+# Update
+sudo apt update
+
+# Installing ca-certificates
+if (test -d ca-certificates)
+then
+  echo "ca-certificates already installed"
+else
+  echo "Installing ca-certificates"
+  sudo apt install -y ca-certificates
+fi
+
+# Installing curl
+if (which curl)
+then
+  echo "curl already installed"
+else
+  echo "Installing curl"
+  sudo apt install -y curl 
+fi
+
+# Creating a keyrings directory 
+if (test -d /etc/bin/keyrings)
+then
+  echo "creating keyrings"
+else
+  echo "Creating keyrings"
+  sudo install -m 0755 -d /etc/apt/keyrings
+fi
+
+# Adding a gpg keyring
+if (stat -c "%a" /etc/bin/keyrings/docker.asc)
```

Comment:
This check would be maybe ok with just `gpg /etc/bin/keyrings/docker.asc` and gpg will check it's validity? I'm not sure but try it.

Reply:
```
chris@docker:~ $ gpg /etc/bin/keyrings/docker.asc
gpg: directory '/home/chris/.gnupg' created
gpg: keybox '/home/chris/.gnupg/pubring.kbx' created
gpg: WARNING: no command supplied.  Trying to guess what you mean ...
gpg: can't open '/etc/bin/keyrings/docker.asc'
```

Reply:
It's saying there is no file at that path. That is a good check. Then you should do an else that creates that file like you had.

Reply:
```
chris@docker:~ $ if ( gpg /etc/bin/keyrings/docker.asc)
then
  echo "gpg keyring already exists"
else
  echo "Installing gpg keyring"
  sudo curl -fsSL https://download.docker.com/linux/debian/gpg -o /etc/apt/keyrings/docker.asc
fi
gpg: WARNING: no command supplied.  Trying to guess what you mean ...
gpg: can't open '/etc/bin/keyrings/docker.asc'
Installing gpg keyring
```